.github/workflows/ci.yml

```diff
@@ -69,19 +81,27 @@ jobs:
             es_version: 7.7.1
           - task: pypkg
             es_version: 7.8.1
+          - task: pypkg
+            es_version: 7.9.3
+          - task: pypkg
+            es_version: 7.10.2
+          - task: pypkg
+            es_version: 7.11.2
+          - task: pypkg
+            es_version: 7.12.1
     steps:
       - name: checkout repository
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v2
```

Comment:
https://github.com/actions/checkout uses `v2` the way some projects use a `stable` tag. For example, if they publish a 2.3.5, at that moment `v2.3.5` and `v2` will point to the same commit.

So this is equivalent to doing something like `>=2.0.0,<3.0.0` in Python